rules/rules-reviewed/camel3/camel2/xml-315-changes.windup.xml

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="xml-315-changes"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            Rules for changes between Camel 3.14 and Camel 3.15
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="camel" versionRange="[3.0,3.15)"/>
+        <targetTechnology id="camel" versionRange="[3.15,)" />
+    </metadata>
+    <rules>
+        <rule id="xml-315-00001">
+            <when>
+                <or>
+                    <project>
+                        <artifact groupId="org.apache.camel" artifactId="{*}"/>
+                    </project>
+                    <dependency groupId="org.apache.camel" artifactId="{*}"/>
+                </or>
+            </when>
+            <perform>
+                <hint title="JDK 8 no longer supported" effort="7" category-id="mandatory" >
+                    <message>JDK 8 no longer supported - In Camel 3.15, JDK 11 or 17 is required.</message>
```

Comment:
@cunningt I don't undertand this rule - how are you detecting that JDK 8 is being used?

Reply:
I'm not sure there's a way to detect the JDK used at runtime.   It's an important enough change that I think displaying the message in all cases is warranted.

Reply:
@cunningt okay, let's change the `category-id` to `potential` then, since we're not sure.

Reply:
Changed category-id to potential